### PR TITLE
UPDATED:User_Form

### PR DESCRIPTION
**Pull request by PrakashVjr**
User form Updated

### CODE REVIEW

File: Prakash/User_reg_form/index.html
Comment (NavinAT) [inline Prakash/User_reg_form/index.html:1]: 1. While hover button the container height should not move or remove button hover style.

https://user-images.githubusercontent.com/45414603/225611502-c80f4047-bd34-4aaf-a59e-7f97b8261089.mp4

2. After the validation the typed value got removed in the form.

https://user-images.githubusercontent.com/45414603/225612262-5bc3f118-0398-4cea-945c-61dd2fc5a109.mp4

3. Remove text box hover style
4. What is the use of these marked two buttons, both buttons navigate to the table page.

image

File: Prakash/User_reg_form/edit.html
Comment (NavinAT) [inline Prakash/User_reg_form/edit.html:1]: 1. Value got removed, after pressing cancel button.

https://user-images.githubusercontent.com/45414603/225614839-73ba3c7c-0fb6-4f07-8d6d-9f7cc95317d8.mp4

2. Age validation is missing (check in create form also).

**Review by NavinAT — CHANGES_REQUESTED** [review]

File: Prakash/User_reg_form/script.js [hunk 1]
```diff
@@ -1,28 +1,90 @@
...
+    var object1=
+            {   
+                fname:fname,email:email,cno:cno,age:age,
+                gender:gender,dob:dob,address:address
+            };
     var arrobj=[];
-    var obj=localStorage.getItem("testJSON");
+    var obj=localStorage.getItem("prakshJSON");
     if(obj)
         {
```
[the remark this answers is not in the document]
Comment (PrakashVjr) [reply Prakash/User_reg_form/script.js:49]: Cleared

File: Prakash/User_reg_form/style1.css [hunk 2]
```diff
@@ -7,92 +7,97 @@ body
...
-    margin-top: 6px;
-    margin-right: 5px;
-    padding:2px 15px;
-    background-image: url(img/g5.jpg);
-    margin-bottom: 10px;
-}
-.options
+    padding-left: 10px;
+}
+.btn{
```
[the remark this answers is not in the document]
Comment (PrakashVjr) [reply Prakash/User_reg_form/style1.css:60]: Cleared

File: Prakash/User_reg_form/style1.css [hunk 3]
```diff
@@ -7,92 +7,97 @@ body
...
+    cursor: pointer;
+}
+.nav_btn:hover
+{
+    border-radius: 20%;
+    border: 1px solid rgba(139, 133, 133, 0.596);
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+}
+input{
+    padding-left: 8px;
```
[the remark this answers is not in the document]
Comment (PrakashVjr) [reply Prakash/User_reg_form/style1.css:78]: Cleared

File: Prakash/User_reg_form/style1.css [hunk 4]
```diff
@@ -111,4 +116,7 @@ th
 hr
 {
     border-top: 2px dashed red;
+}
+tr:hover{
+    background-color: white;
```
[the remark this answers is not in the document]
Comment (PrakashVjr) [reply Prakash/User_reg_form/style1.css:121]: Cleared

File: Prakash/User_reg_form/index.html
Comment (PrakashVjr) [reply Prakash/User_reg_form/index.html:1]: Cleared

File: Prakash/User_reg_form/edit.html
Comment (PrakashVjr) [reply Prakash/User_reg_form/edit.html:1]: Cleared

**Pull request merged**